.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,14 @@
+name: Question
+description: Ask a question
+#title: ""
+labels: ["question"]
```

Comment:
I think it would be better if people used [Discussions](https://github.com/input-output-hk/cardano-js-sdk/discussions) to ask questions